test/src/e2e_vm_tests/test_programs/test_fuel_coin_abi/Forc.toml

```diff
@@ -1,8 +1,8 @@
 [project]
 author  = "Fuel Labs <contact@fuel.sh>"
+entry = "main.sw"
 license = "Apache-2.0"
 name = "test_fuel_coin_abi"
-entry = "main.sw"
 
 [dependencies]
-std = { git = "http://github.com/FuelLabs/sway-lib-std", version = "v0.0.1" }
+std = { git = "http://github.com/FuelLabs/sway-lib-std" }
```

Comment:
are we deciding to stop pinning deps in tests?

Reply:
Maybe I misunderstood. In a slack conversation, I thought that you had said we should be using the latest stdlib, which removing the version info _should_ accomplish?
https://fuellabs.slack.com/archives/C01KBNGSWGJ/p1643224384043300?thread_ts=1643224182.042400&cid=C01KBNGSWGJ

Reply:
I'm happy to modify this if we want to keep the version pinned to the current latest version, and have the ongoing maintenance chore of bumping the versions for our tests as we go... But for now there will be pretty rapid iteration on the stdlib so maybe no pinned version yet?

Reply:
I think we wanted to intentionally have to do the work of version bumping, otherwise a community contribution to the standard library could be breaking and halt all sway development (the CI would fail) until the standard library is updated to the new API. what do you think?